test/api/RequestObject/ingestion_json.py

```diff
@@ -212,10 +103,9 @@ def verify_ingestion_attribute(self, actual_results, expected_results, attribute
   def verify_array(self, actual_array, expected_array):
     self.assertIsNotNone(actual_array)
     self.assertIsNotNone(expected_array)
-    #self.assertEquals(len(actual_array), len(expected_array), 'The arrays have a different size')
 
   """
-  Below SQL statements will get article id from ambra db given doi  example 10.1371/journal.pone.0155391
+  Below SQL statements will query ambra article table given doi  example 10.1371/journal.pone.0155391 for article id
```

Comment:
PEP-8 and other style guides recommends putting comments as docstrings, that is inside each function/method. State general purpose, parameters (name and data type and an exaplanation) and also information on what the methods returns. Will put only one note on this but it applies for all your changes.